css/style.css

```diff
@@ -3294,6 +3332,31 @@ span a {
     width: 670px;
 }
 
+.mega {
+    column-count: 3;
```

Comment:
Why column-count if already described with grid-template-columns?

Reply:
when I am doing work on this section at that point i use column-count but with that I didn't get the result i want so i use grid-template-column but i forgot to remove column-count at that particular time. but now i fix that one issue and already push on git.